services/docker-compose-ops.yml

```diff
@@ -83,7 +83,7 @@ services:
         user_notifications:${REDIS_HOST}:${REDIS_PORT}:4,
         announcements:${REDIS_HOST}:${REDIS_PORT}:5,
         distributed_identifiers:${REDIS_HOST}:${REDIS_PORT}:6,
-        distributed_caches:${REDIS_HOST}:${REDIS_PORT}:7
+        deferred_tasks:${REDIS_HOST}:${REDIS_PORT}:7
```

Comment:
we do not use "distributed_caches"?

Reply:
no, that PR never made it in. I've changed it since it was not used